cloudbuild.yaml

```diff
@@ -7,12 +7,24 @@ steps:
     waitFor: ['-']
     timeout: 10m
     args: ['pull', 'gcr.io/oak-ci/oak:latest']
+  # Pull Android Docker image.
```

Comment:
I would keep the Android stuff at the end, in fact perhaps adds an extra line between the current steps and the android steps. I'm thinking we may want to split the triggers into two separate cloudbuild files in the short future. WDYT?

Reply:
Done